migrations/Version20251212143025.php

```
@@ -0,0 +1,36 @@
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20251212143025 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Removes legacy page';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)